Vagrantfile

```diff
@@ -0,0 +1,37 @@
+control_node_cores = 1
+control_node_mem = 4
+control_node_ip = "192.168.60.2"
+
+worker_nodes = 2
+worker_node_cores = 2
+worker_node_mem = 6
+worker_node_ip_base = "192.168.61"
+
+
+Vagrant.configure("2") do |config|
+    config.vm.box = "bento/ubuntu-24.04"
+    config.vm.box_version = "202404.26.0"
+
+    # I know this is bad but I cannot get it to work with SSH when running vagrant up.
+    config.ssh.password = "vagrant"
```

Comment:
Please have another look at setting up ssh. This is indeed bad.

Reply:
I have already spent like 5h trying to set up ssh. This way, we can still connect via ssh, it only uses the password during setup. In the assignment description, I cannot find any requirement related to using ssh or passwords on setup so I dont think it is worth it.

Reply:
Ok, I approve